src/components/Shared/Table.js

```diff
@@ -485,7 +485,12 @@ const Table = ({
             },
             '&:focus': {
               outline: 'none'
-            }
+            },
+            ...(!params.colDef?.wrapText && {
```

Comment:
why do you need to check for wrapText? cannot be added directly?

Reply:
wraptext is feature in ag grid and it effects on this fixed